public/javascripts/directives/calc-listitem.js

```diff
@@ -0,0 +1,25 @@
+angular.module('undp-ghg-v2')
+    .directive('calcItem', function() {
+        return {
+            restrict: 'E',
+            scope: {
+                subItem: "=",
+                activity: "=",
+                selected: "="
+            },
+            template: '<div layout="row" class="subitem" ui-tree-handle>' +
+                 '<input type="checkbox" ng-model="iselected.item" ng-change="calculate()" />' +
+                 '<span flex>{{subItem.ac_activity.ac_name}} - {{subItem.ca_category.ca_code_name}} - {{subItem.ga_gas.ga_gas_name}}</span>' +
+                 '<span flex ng-show="iselected.item" style="color: #000000;">{{calculated_value}} {{subItem.un_unit.un_unit_symbol}}</span></div>',
```

Comment:
@gilgad `{{subItem.un_unit.un_unit_symbol}}` is actually not the correct symbol for the amount of gas emitted from the activity.  This value is actually the emission factor unit.  The unit can actually be derived but I don't think it would be wise to derive it as it might be error prone.... my suggestion is to allow the user to select the unit for the outcome.